content/sensu-core/2.0/reference/checks.md

```diff
@@ -312,6 +312,20 @@ description  | If the event is to be silenced.
 type         | boolean
 example      | {{< highlight shell >}}"silenced": false{{< /highlight >}}
 
+|output_metric_format    |      |
+-------------|------
+description  | The metric format generated by the check command. Sensu supports the following metric formats: <br>`nagios_perfdata` ([Nagios Performance Data][nagios]) <br>`graphite_plaintext` ([Graphite Plaintext Protocol][graphite]) <br>`influxdb_line` ([InfluxDB Line Protocol][influx]) <br>`opentsdb_line` ([OpenTSDB Data Specification][open])<br><br>When a check includes an `output_metric_format`, Sensu will extract the metrics from the check output and add them to the [event data][24] within the `metrics` scope. For more information about extracting metrics using Sensu, see the [guide][23]. 
+required     | false (true if `output_metric_handlers` is configured)
```

Comment:
I don't think `true if output_metric_handlers is configured` is actually enforced at the source level, although it is a good guideline. For example, one _can_ set `output_metric_handlers` and not specify `output_metric_format` however, metrics would not actually be extracted. Let me know if that makes sense?